docs/source/DeveloperGuide/Internals/Core/Authentification.rst

```diff
@@ -0,0 +1,35 @@
+==================================
+Authentification and authorization
+==================================
+When a user is calling DIRAC he need to be identified. When a client open a connection every thing you need is stored in BaseTransport object (DIRAC.Core.DISET.private.transport.BaseTransport). DIRAC use the handshake to read certificates, extract informations and store them in a dictionnary so you can use these informations easily. Here an example of possible dictionnary::
```

Comment:
`needs`. tu oublies toujours le S a la troisieme personne, je te laisse relire le reste

Reply:
je ne comprends pas cette phrase `When a client open a connection every thing you need is stored in BaseTransport object (DIRAC.Core.DISET.private.transport.BaseTransport)`. 
Tu peux faire un lien vers la code documentation avec "py:class:`~DIRAC.Core.DISET.private.transport.BaseTransport.BaseTransport`"